LayoutTests/platform/mac-wk1/TestExpectations

```diff
@@ -2594,28 +2594,19 @@ imported/w3c/web-platform-tests/websockets/interfaces/WebSocket/bufferedAmount/b
 imported/w3c/web-platform-tests/websockets/interfaces/WebSocket/bufferedAmount/bufferedAmount-unicode.html?wss [ Pass Failure ]
 
 #webkit.org/b/248537 Batch mark expectations for Ventura test failures
-[ Ventura+ arm64 ] svg/transforms/transformed-text-fill-pattern.html [ ImageOnlyFailure ]
-[ Ventura+ ] media/accessibility-describes-video.html [ Crash ]
-[ Ventura+ ] media/modern-media-controls/pip-support/pip-support-click.html [ Pass Crash ]
-[ Ventura+ ] media/audio-playback-restriction-removed-track-enabled.html  [ Crash ]
-[ Ventura+ ] media/modern-media-controls/media-controller/media-controller-auto-hide-rewind-with-mouse-enter.html  [ Crash ]
-[ Ventura+ ] media/modern-media-controls/media-controls/media-controls-display-above-captions.html  [ Crash ]
-[ Ventura+ ] media/track/track-delete-during-setup.html  [ Crash ]
-[ Ventura+ ] media/track/track-in-band-cues-added-once.html  [ Crash ]
-[ Ventura+ ] media/track/track-in-band.html  [ Crash ]
-[ Ventura+ ] media/track/track-long-captions-file.html  [ Crash ]
-imported/w3c/web-platform-tests/secure-contexts/basic-popup-and-iframe-tests.html [ Pass Failure ]
-[ Ventura+ ] http/wpt/webaudio/audioworklet-addModule-cors.sub.https.html
+[ arm64 ] svg/transforms/transformed-text-fill-pattern.html [ ImageOnlyFailure ]
+media/modern-media-controls/media-controller/media-controller-auto-hide-rewind-with-mouse-enter.html [ Pass Crash ]
+media/track/track-in-band-cues-added-once.html [ Pass Crash ]
```

Comment:
Not a blocker for landing this, but a some point soon we should make certain that there are bugs filed at least for crashes, if not for all of the regressions.